test_framework/terraform/aws/rockylinux/user-data-scripts/provision_rke.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
this is different from the other template file like .tpl. Does it matter? If not, probably we can change all of them to .sh instead.

Reply:
fixed, Thank you.